File: app/src/main/java/com/geovanni/banks/bussiness/entityDao/interfaces/IServiceBanksDao.java
package com.geovanni.banks.bussiness.entityDao.interfaces;

import androidx.room.Dao;
import androidx.room.Insert;
import androidx.room.Query;

import com.geovanni.banks.bussiness.models.ServiceBanksResponse;

import java.util.List;

@Dao
public interface IServiceBanksDao {
    @Query("SELECT * FROM ServiceBanksResponse")
    List<ServiceBanksResponse> getAll();

    @Insert
    void insert(ServiceBanksResponse bank);
}

File: app/src/main/java/com/geovanni/banks/bussiness/entityDao/interfaces/IServiceBanksDao.kt
package com.geovanni.banks.bussiness.entityDao.interfaces

import androidx.room.Dao
import androidx.room.Insert
import androidx.room.Query
import com.geovanni.banks.bussiness.models.ServiceBanksResponse

@Dao
interface IServiceBanksDao {
    @get:Query("SELECT * FROM ServiceBanksResponse")
    val all: List<ServiceBanksResponse?>?

    @Insert
    fun insert(bank: ServiceBanksResponse?)
}